### PR TITLE
docs (develop + protocol + validate) sectional updates

### PR DESCRIPTION
**Pull request by masterfung**
Updated progress to the Develop and some layout improvements.

### CODE REVIEW

File: docs/develop/tools/contract-verifications.md [hunk 1]
```diff
@@ -7,3 +7,18 @@ Contract verification refers to the process of auditing and verifying the code o
 
 It is typically performed by independent auditors, security experts, or automated tools. The process involves a thorough analysis of the contract code to identify potential issues, such as bugs, security vulnerabilities, or potential attacks. This can involve both manual and automated testing to ensure that the code meets certain standards and best practices.
 
+## [EScan](https://escan.live/)
```
Comment (danburck) [inline docs/develop/tools/contract-verifications.md:10]: I saw that they spell it differently, when you look at the tab title on https://escan.live/

```suggestion
## [Escan](https://escan.live/)
```

File: docs/develop/tools/contract-verifications.md [hunk 2]
```diff
@@ -7,3 +7,18 @@ Contract verification refers to the process of auditing and verifying the code o
...
+
+There are several block explorers serving the Evmos network and EScan is a service that allows block exploration as well as contract verification. For more information about the different block explorers, please proceed to our [block explorers's page](./explorers).
```
Comment (danburck) [inline docs/develop/tools/contract-verifications.md:12]:
```suggestion
Escan is a service that allows block exploration as well as contract verification. For more information about the different block explorers, please proceed to our [block explorers's page](./explorers).
```

File: docs/develop/tools/contract-verifications.md [hunk 3]
```diff
@@ -7,3 +7,18 @@ Contract verification refers to the process of auditing and verifying the code o
...
+
+[Link](https://escan.live/verifyContract) to verify and publish the contract source code.
+
+![escan-contract.png](/img/escan-contract.png)
+
+## [Mintscan](https://www.mintscan.io/evmos/evm)
+
+![mintscan-evm-contracts.png](/img/mintscan-evm-contracts.png)
```
Comment (danburck) [inline docs/develop/tools/contract-verifications.md:20]: keep consistent order of images and descriptions for each tool, e.f. for Escan the image comes after the description.

**Review by danburck — COMMENTED** [review]
@masterfung I'm not sure what kind of review you are expecting if there are so many different changes and some of them are still work in progress. Would be nice to have smaller PRs for specific sections, to make sure they are right.

I left some comments, but didn't review everything

File: docs/develop/build-a-dApp/run-a-node/installation.md [hunk 4]
```diff
@@ -33,12 +35,12 @@ export PATH=$PATH:$(go env GOPATH)/bin
 ## Install Binaries
 
 :::tip
-The latest {{ $themeConfig.project.name }} [version](https://github.com/evmos/evmos/releases) is `{{ $themeConfig.project.binary }} {{ $themeConfig.project.latest_version }}`
+The latest <ProjectValue keyword='name' /> [version](https://github.com/evmos/evmos/releases) is `{{ $themeConfig.project.binary }} {{ $themeConfig.project.latest_version }}`
```
[the remark this answers is not in the document]
Comment (masterfung) [reply docs/develop/build-a-dApp/run-a-node/installation.md:38]: We can address this once we have all our docs in. I will refrain from changing them until then.

**Comment by masterfung** [comment]
@danburck please re-review

**Review by danburck — APPROVED** [review]
Overall LGTM

**Pull request merged**